pkg/utils/common.go

```diff
@@ -51,8 +54,9 @@ const (
 )
 
 const (
-	PlacementFieldManagerName    = "cluster-placement-controller"
-	MCControllerFieldManagerName = "member-cluster-controller"
+	PlacementFieldManagerName          = "cluster-placement-controller"
```

Comment:
i'm thinking of getting rid of the utils package as the utils is not a good package name and it's too generic.

We can always create the package, like FieldManager, etc.